### PR TITLE
2021-12-03-xlm_roberta_large_token_classification_ner_id

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """Nama saya Sarah dan saya tinggal di London."""
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md:63]: Can we get a bigger example, with more words and more detected entities, that showcase all the capabilities of the model? @murat-gunay  THank you!

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can we get a bigger example, with more words and more detected entities, that showcase all the capabilities of the model? @murat-gunay THank you!

File: docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md]: @josejuanmartinez 
Thanks for your feedback. I changed the text and output result to get more entities but this is not a demo just a model page. the purpose of the scripts and example text is just to show customers how to use this model in coding.

File: docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is imported from `Hugging Face-models`. It is the Indonesian fine-tuned of "xlm-roberta-large" language model.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md:20]: This model has been imported from `Hugging Face`. It's been trained using a `xlm-roberta-large` finetuned model on Indonesian texts.

File: docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+layout: model
+title: Indonesion NER Model
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-xlm_roberta_large_token_classification_ner_id.md:3]: Typo: `Indonesian`

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
@murat-gunay Can you apply this changes? Thank you!

**Review by josejuanmartinez — APPROVED** [review]
Looks good, thank you!

**Pull request merged**